### ISSUE TITLE
Update Test or add a new Test for testing Alt Text of images

### ISSUE THREAD

**Issue by clapierre** [daisy/epub-accessibility-tests#61]
I think this should be a new test as some Reading Systems may have a setting to include alt text descriptions of images read using the Read Aloud feature of the system.  We should explicitly call out this in this new test.

**Comment by ways2read** [comment]
On the call 17 October we decided to close this issue as it is covered in the existing test,
